vip-helpers/vip-wp-cli.php

```diff
@@ -2,23 +2,12 @@
 
 class WPCOM_VIP_CLI_Command extends WP_CLI_Command {
 
-	/*
+	/**
 	 *  Clear all of the caches for memory management
 	 */
 	protected function stop_the_insanity() {
-		global $wpdb, $wp_object_cache;
-
-		$wpdb->queries = array(); // or define( 'WP_IMPORTING', true );
-
-		if ( !is_object( $wp_object_cache ) )
-			return;
-
-		$wp_object_cache->group_ops = array();
-		$wp_object_cache->memcache_debug = array();
-		$wp_object_cache->cache = array();
-
-		if ( is_callable( $wp_object_cache, '__remoteset' ) )
-			$wp_object_cache->__remoteset(); // important
+		wpcom_vip_flush_db_query_log();
+		wpcom_vip_flush_object_cache();
```

Comment:
Moved this logic to `vip-caching.php`, as it's easier to use there (as this is only for classes inheriting `WPCOM_VIP_CLI_Command`) and because it makes more sense.